doc/source/whatsnew/v1.5.0.rst

```diff
@@ -489,6 +489,7 @@ Period
 ^^^^^^
 - Bug in subtraction of :class:`Period` from :class:`PeriodArray` returning wrong results (:issue:`45999`)
 - Bug in :meth:`Period.strftime` and :meth:`PeriodIndex.strftime`, directives ``%l`` and ``%u`` were giving wrong results (:issue:`46252`)
+- Bug in :meth:`Period.strftime` and :meth:`PeriodIndex.strftime`, Unicode decoding error when a locale-specific directive is used (:issue:`46319`)
```

Comment:
", Unicode decoding error" -> " raising \`UnicodeDecodeError\`"

"is used" -> "was passed"

Reply:
ok